### PR TITLE
Update readme

### LINKED ISSUE #1: publish ansible setup with tutorial how to setup OVH bare-metal infra1 machine
Please add ansible setup to install a fully featured SGX dev machine
Also, describe how to order a machine on OVH and use ansible to set it up

### PR DESCRIPTION
**Pull request by haerdib**
Not fully tested but should be somewhat in the right direction.

### CODE REVIEW

**Review by brenzi — APPROVED** [review]
looks very nice. Looking forward to playing it through at the next occasion

**Review by brenzi — APPROVED** [review]

**Comment by mosonyi** [comment]
sgxhosts file should contain the hostname and this info is not mentioned in the doc

File: README.md [hunk 1]
```diff
@@ -1,2 +1,138 @@
...
+    2. Execute the script according to [Ansible Script Execution](https://github.com/integritee-network/sgx-setup/tree/add-readme#ansible-script-execution).
+3. Install gcc-tools by activating the [role-andrewrothstein.gcc-toolbox](https://github.com/integritee-network/sgx-setup/blob/main/sgx-ansible/DevelopmentServersSGX.yml#L20) (don't forget to recomment previously executed roles) and execute it, just like before.
+4. Finally activate the [role-intel-sgx](https://github.com/integritee-network/sgx-setup/blob/main/sgx-ansible/DevelopmentServersSGX.yml#L22) and execute it as well. This might take some time (30 minutes on fast machines). In case the smoke tests have passed: Congratulation: You have successfully set up an sgx machine :)
+ In case they did not, but the script executed without any errors, take a look at section [Check if sgx is enabled in BIOS](https://github.com/integritee-network/sgx-setup/tree/add-readme#check-if-sgx-is-enabled). If that is not the error, create an [issue](https://github.com/integritee-network/sgx-setup/issues/new) so we can try to help you out.
+5. Seting up Users:
```
Comment (mosonyi) [inline README.md]: typo: setting

**Review by mosonyi — CHANGES_REQUESTED** [review]
Please fix the typo

File: README.md (hunk 1 above)
Comment (haerdib) [reply README.md]: Should be fixed now :)

**Review by mosonyi — APPROVED** [review]
LGTM

**Pull request merged**